elements/pfe-cta/src/pfe-cta.scss

```diff
@@ -120,6 +122,7 @@ $variables: (
 
   :host([pfe-priority]) & {
     text-align: center;
+    font-size: #{pfe-local($cssvar: FontSize--priority, $fallback: #{pfe-var(font-size)})};
   }
 }
```

Comment:
@chrisdo1 sorry i just noticed, i think we also need this on the disabled ctas too
```
:host([aria-disabled="true"]) {
    font-size: #{pfe-local($cssvar: FontSize--priority, $fallback: #{pfe-var(font-size)})};
}
```